xtra-libp2p/tests/basic.rs

```diff
@@ -312,43 +312,7 @@ async fn falls_back_to_next_protocol_if_unsupported() {
 
 #[tokio::test]
 async fn disconnect_after_too_many_open_substream_failures() {
-    let alice = make_node([]);
-    let bob = make_node([]);
-
-    let port = rand::random::<u16>();
-    let alice_listen = format!("/memory/{port}").parse::<Multiaddr>().unwrap();
-
-    alice
-        .endpoint
-        .send(ListenOn(alice_listen.clone()))
-        .await
-        .unwrap();
-
-    let alice_peer_id = &alice.peer_id;
-
-    let alice_connected_peers = alice
-        .subscriber_stats
-        .send(GetConnectedPeers)
-        .await
-        .unwrap();
-    assert!(alice_connected_peers.is_empty());
-
-    let bob_connected_peers = alice
-        .subscriber_stats
-        .send(GetConnectedPeers)
-        .await
-        .unwrap();
-    assert!(bob_connected_peers.is_empty());
-
-    bob.endpoint
-        .send(Connect(
-            format!("/memory/{port}/p2p/{alice_peer_id}")
-                .parse()
-                .unwrap(),
-        ))
-        .await
-        .unwrap()
-        .unwrap();
+    let (alice, bob, _) = alice_and_bob([], []).await;
```

Comment:
hahaha, I had this initially but it did not work. Because of my implementation 😅
Then I never changed it back :D
Thanks!